custom_dc/index.md

```diff
@@ -55,22 +55,19 @@ For the following use cases, a custom Data Commons instance is not necessary:
 
 ## System overview
 
-Essentially, a custom Data Commons instance is a mirror of the public Data Commons, that runs in a [Docker](http://docker.com) container hosted in the cloud. In the browsing tools, the custom data appears alongside the base data in the list of variables. When a query is sent to the custom website, the Mixer server fetches both the custom and base data to provide multiple visualizations. At a high level, here is a conceptual view of a custom Data Commons instance:
+Essentially, a custom Data Commons instance is a mirror of the public Data Commons, that runs in [Docker](http://docker.com) containers hosted in the cloud. In the browsing tools, the custom data appears alongside the base data in the list of variables. When a query is sent to the custom website, a Data Commons server fetches both the custom and base data to provide multiple visualizations. At a high level, here is a conceptual view of a custom Data Commons instance:
 
 ![setup1](/assets/images/custom_dc/customdc_setup1.png){: height="450" }
 
-The custom Data Commons Docker container consists of the following components:
+A custom Data Commons instance uses custom data that you provide as raw CSV files. An importer script converts the CSV data into the Data Commons format and stores this in a SQL database. For local development, we provide a lightweight, open-source [SQLite](http://sqlite.org) database; for production, we recommend that you use [Google Cloud SQL](https://cloud.google.com/sql/).
 
-- A [Nginx reverse proxy server](https://www.nginx.com/resources/glossary/reverse-proxy-server/), which routes incoming requests to the web or API server
-- A Python-Flask web server, which handles interactive requests from users
-- An Python-Flask NL server, for generating embeddings and serving natural language queries
-- A Go Mixer, also known as the API server, which serves programmatic requests using Data Commons APIs. The SQL query engine is built into the Mixer, which sends queries to both the local and remote data stores to find the right data. If the Mixer determines that it cannot fully resolve a user query from the custom data, it will make an REST API call, as an anonymous "user" to the base Data Commons Mixer and data.
+In addition to the data, a custom Data Commons instance consists of two Docker containers: one with the core services that serve the data and website; and one with additional utilities for managing and loading custom data and embeddings used for natural-language processing. 
```

Comment:
Perhaps change "additional utilities" to just "utilities"?

Reply:
Yup.